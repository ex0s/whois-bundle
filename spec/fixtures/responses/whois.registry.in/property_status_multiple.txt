Access to .IN WHOIS information is provided to assist persons in 
determining the contents of a domain name registration record in the 
.IN registry database. The data in this record is provided by 
.IN Registry for informational purposes only, and .IN does not 
guarantee its accuracy.  This service is intended only for query-based 
access. You agree that you will use this data only for lawful purposes 
and that, under no circumstances will you use this data to: (a) allow, 
enable, or otherwise support the transmission by e-mail, telephone, or 
facsimile of mass unsolicited, commercial advertising or solicitations 
to entities other than the data recipient's own existing customers; or 
(b) enable high volume, automated, electronic processes that send 
queries or data to the systems of Registry Operator, a Registrar, or 
Afilias except as reasonably necessary to register domain names or 
modify existing registrations. All rights reserved. .IN reserves 
the right to modify these terms at any time. By submitting this query, 
you agree to abide by this policy. 

Domain ID:D3271170-AFIN
Domain Name:VIDEOGRATIS.IN
Created On:27-Jan-2009 05:01:05 UTC
Last Updated On:01-Jul-2010 12:55:17 UTC
Expiration Date:27-Jan-2011 05:01:05 UTC
Sponsoring Registrar:GoDaddy.com Inc. (R101-AFIN)
Status:CLIENT DELETE PROHIBITED
Status:CLIENT RENEW PROHIBITED
Status:CLIENT TRANSFER PROHIBITED
Status:CLIENT UPDATE PROHIBITED
Registrant ID:CR51214064
Registrant Name:claudio spada
Registrant Organization:sirismedia
Registrant Street1:foro buonaparte 69
Registrant Street2:
Registrant Street3:
Registrant City:milano
Registrant State/Province:italy
Registrant Postal Code:20121
Registrant Country:AX
Registrant Phone:+91.03902861317
Registrant Phone Ext.:
Registrant FAX:
Registrant FAX Ext.:
Registrant Email:domini@siris.com
Admin ID:CR51214074
Admin Name:claudio spada
Admin Organization:sirismedia
Admin Street1:foro buonaparte 69
Admin Street2:
Admin Street3:
Admin City:milano
Admin State/Province:italy
Admin Postal Code:20121
Admin Country:AX
Admin Phone:+91.03902861317
Admin Phone Ext.:
Admin FAX:
Admin FAX Ext.:
Admin Email:domini@siris.com
Tech ID:CR51214069
Tech Name:claudio spada
Tech Organization:sirismedia
Tech Street1:foro buonaparte 69
Tech Street2:
Tech Street3:
Tech City:milano
Tech State/Province:italy
Tech Postal Code:20121
Tech Country:AX
Tech Phone:+91.03902861317
Tech Phone Ext.:
Tech FAX:
Tech FAX Ext.:
Tech Email:domini@siris.com
Name Server:NS1.SLICEHOST.NET
Name Server:NS2.SLICEHOST.NET
Name Server:NS3.SLICEHOST.NET
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 

