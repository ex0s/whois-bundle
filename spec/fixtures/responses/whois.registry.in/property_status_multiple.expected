#status
  %s == ["CLIENT DELETE PROHIBITED", "CLIENT RENEW PROHIBITED", "CLIENT TRANSFER PROHIBITED", "CLIENT UPDATE PROHIBITED"]
